ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -60,7 +62,7 @@
   private BlobStoreCompactor compactor;
   private PersistentIndex index;
   private BlobStoreStats blobStoreStats;
-  private boolean started;
+  private volatile boolean started;
```

Comment:
This wasn't so much for the DiskSpaceAllocator, but because shutdown could possibly be called while requests are coming in. Maybe this isn't a situation we have to worry about because replication and request handlers are shut down before the blob stores are?